packages/ui/src/components/TokenMedia/MediaPlayButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC, useState, useEffect, RefObject } from 'react'
+import { Button } from '../../primitives'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPlay, faPause } from '@fortawesome/free-solid-svg-icons'
+
+type MediaPlayButtonProps = {
+  mediaRef: RefObject<HTMLAudioElement> | RefObject<HTMLVideoElement>
+}
+
+const MediaPlayButton: FC<MediaPlayButtonProps> = ({ mediaRef }) => {
+  const [playing, setPlaying] = useState(false)
+
+  useEffect(() => {
+    if (mediaRef.current) {
+      mediaRef.current.onplaying = () => {
+        setPlaying(true)
+      }
+      mediaRef.current.onpause = () => {
+        setPlaying(false)
+      }
+    }
+    return () => {
+      if (mediaRef.current) {
+        mediaRef.current.onplaying = null
+        mediaRef.current.onpause = null
+      }
+    }
+  }, [mediaRef])
+
+  return (
+    <Button
+      css={{
+        zIndex: 5,
+        position: 'absolute',
```

Comment:
feel like components should really position themselves. think it would be better to do this while laying it out. not a big deal here since it's basically only going to be used in the one context

Reply:
Right, we decided that the container's are resizable / open to css changes. It's difficult to allow developers to customize this button css without exposing another property and making the props interface more complex. Maybe in the future we can circle back to it when we get more feedback.